sm8650.c

```diff
@@ -144,317 +144,317 @@ static struct debug_mux video_cc = {
 
 static struct measure_clk sm8650_clocks[] = {
 	/* GCC entries */
-	{ "gcc_aggre_noc_pcie_axi_clk", &gcc.mux, 0x3f },
```

Comment:
I usually like C99. However in this case struct initialisation becomes less readable.

Reply:
Well either that or we have to fill out all magic fields to make the "struct is only partially initialized" warning (forgot the name) satisfied

Reply:
Or disable the warning